classes/doi/DAO.inc.php

```diff
@@ -35,7 +35,7 @@ class DAO extends \PKP\doi\DAO
      */
     public function getAllDepositableSubmissionIds(Context $context): Collection
     {
-        $enabledDoiTypes = $context->getData(Context::SETTING_ENABLED_DOI_TYPES);
+        $enabledDoiTypes = $context->getData(Context::SETTING_ENABLED_DOI_TYPES) ?? [];
```

Comment:
Nothing that needs to happen now, but I see this line a lot. I wonder if this deserves a helper function on the `Context` object that will always return an array:

```php
public function getEnabledDoiTypes(): array
```